pom.xml

```
@@ -106,6 +107,11 @@ Wisconsin-Madison and University of Konstanz.</license.copyrightOwners>
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-table</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.scijava</groupId>
+			<artifactId>scijava-optional</artifactId>
+			<version>1.0.0</version>
```

We should make this a version property, i.e.:
```suggestion
			<version>${scijava-optional.version}</version>
```